test/fixtures/systemd-socket-unit.yml

```diff
@@ -0,0 +1,43 @@
+# syntax=local/dalec/frontend
```

Comment:
I've deleted the systemd-* test fixtures for now. In the future I think it would be nice to have things like that as integration tests, which I think would require us to actually start containers built by dalec and exec commands in them for testing. How do you feel about that?

Reply:
I agree it would be good to have some tests around this, I'm not sure what that looks like just yet.

There is some support for manually managing the lifecycle of a container with buildkit, but its fairly minimal functionality and, depending on what we end up needing, could require a lot of extra stuff to make it do what we need, including managing our own buildkit instance so we can have the right security entitlements for running systemd.

Reply:
Looks like the files are still here as well.

Reply:
Whoops forgot to delete the actual fixtures